pkg/session_recording/recordings.jsx

```diff
@@ -304,8 +304,8 @@
             return (
                 <div>
                     <div className="content-header-extra">
-                        <table class="form-table-ct">
-                            <tr>
+                        <table className="form-table-ct">
```

Comment:
For now, I've just copied your subjects, but I'll try to be more precise in future, thank you.